src/operator/tensor/matrix_op.cc

```diff
@@ -423,24 +423,27 @@ NNVM_REGISTER_OP(_backward_clip)
 NNVM_REGISTER_OP(repeat)
 .describe(R"code(Repeat elements of an array.
 
-In default, ``repeat`` flatten the input array into 1-D and then repeat the
+By default, ``repeat`` flattens the input array into 1-D and then repeats the
 elements::
 
   x = [[ 1, 2],
        [ 3, 4]]
 
   repeat(x, repeats=2) = [ 1.,  1.,  2.,  2.,  3.,  3.,  4.,  4.]
 
-We can also choose a particular axis to repeat, in which a negative axis is
-interpreted counting from the backward::
+The parameter axis specifies the axis along which to perform repeat::
 
   repeat(x, repeats=2, axis=1) = [[ 1.,  1.,  2.,  2.],
                                   [ 3.,  3.,  4.,  4.]]
 
-  repeat(x, repeats=2, axis=-1) = [[ 1.,  2.],
-                                   [ 1.,  2.],
-                                   [ 3.,  4.],
-                                   [ 3.,  4.]]
+  repeat(x, repeats=2, axis=0) = [[ 1.,  2.],
+                                  [ 1.,  2.],
+                                  [ 3.,  4.],
+                                  [ 3.,  4.]]
+
+  repeat(x, repeats=2, axis=-1) = [[ 1.,  1.,  2.,  2.],
```

Comment:
I think having a example for the negative indexing case might help understanding. what do you think? does it look redundant?